app/controllers/api/v1/search_controller.rb

```diff
@@ -619,6 +632,29 @@ def self.match_studies_by_facet(query_results, search_facets)
         matches
       end
 
+      # Simplify TDR results to be mappable for the UI badges for facet search
+      def self.simplify_TDR_facet_search_results(query_results, search_facets)
```

Comment:
Ruby convention is to not use capitals in method names - those are typically reserved for class/module names.
```suggestion
      def self.simplify_tdr_facet_search_results(query_results, search_facets)
```